### DISCUSSION TITLE
Equivalent feature to `<Plug>Lightspeed_;_sx`?

### DISCUSSION THREAD

**Discussion by goingtosleep** [ggandor/leap.nvim#115] [Q&A] [answered]
Hi, I have a question regarding the __bidirectional__ search. With `lightspeed`, I can use `<Plug>Lightspeed_omni_s`, type the 2 chars, then call `<Plug>Lightspeed_;_sx` to jump to the next label (below cursor position), or `<Plug>Lightspeed_,_sx` to go to the previous one. I'm wondering if `leap` provides a similar feature?

Thank you!

**Comment by ggandor** [comment, marked as answer]
Short answer: no. What happens when you omni-search, and then `;`, is that you simply invoke `omni_s` for registering the search pattern, and then you exit immediately, and initiate a new, directional search with `;`, using the last search pattern. It's kind of a trick. But leap doesn't have an equivalent `;` key, that can initiate a motion itself. I'm wondering about the actual usefulness of the method you described. If you don't want to use labels, and you _do_ want to follow a direction anyway, then why not just use `s` and `S` in the first place?

**Reply by goingtosleep** [reply]
Alright, I understand now. Personally, I find it easier to just hit a key to use `omni_s` instead of having to check the direction first, but I'll give the directional ones a try too. Thank you for your response 😊
